sunpy/map/maputils.py

```diff
@@ -60,11 +61,9 @@ def all_corner_coords_from_map(smap):
     """
     Returns the coordinates of the pixel corners in a map.
     """
-    nx, ny = smap.data.shape
-    xpix = np.arange(0, nx + 1, 1) - 0.5
-    ypix = np.arange(0, ny + 1, 1) - 0.5
```

Comment:
Incidentally, I'll note that `nx`, `ny`, `xpix`, and `ypix` were all incorrectly named – the `x` and `y` are swapped – but `meshgrid()` is also used weirdly, which cancels out the swap (from #4776).  Bad @dstansby! =P

Reply:
Yikes, no, it doesn't cancel out.  The existing code is bugged!

Reply:
😬 thanks for fixing this